src/components/Notifications/EditableActionRow/RecipientForm.tsx

```diff
@@ -42,16 +43,18 @@ export const RecipientForm: React.FunctionComponent<RecipientFormProps> = props
                 selected={ props.action.recipient }
                 onClear={ props.recipientOnClear }
                 onOpenChange={ props.onOpenChange }
+                validated={ props.error }
```

Comment:
This could be changed to !!props.error (to transform it to boolean) (need to update the other and the props)